File: leetcode/136.cpp
#include <bits/stdc++.h>

using namespace std;

typedef long long ll;

class Solution
{
public:
    int singleNumber(vector<int> &nums)
    {
        int ans = 0;
        for (int e : nums)
        {
            ans ^= e;
        }

        return ans;
    }
};

File: python/leetcode/136.py
from typing import List


class Solution:
    def singleNumber(self, nums: List[int]) -> int:
        xor = 0

        for n in nums:
            xor ^= n

        return xor

if __name__ == "__main__":
    nums = [4,1,2,1,2]
    print(Solution().singleNumber(nums))